recipe/meta.yaml

```diff
@@ -10,20 +10,21 @@ source:
   sha256: 0a04f1d7d35a06cdff506593330d4414ea242c9172498ce191f5742eac499e17
 
 build:
-  skip: true  # [win or python_impl == 'pypy']
-  number: 1
+  # skip: true  # [python_impl == 'pypy']
```

Comment:
Should we drop this or should it be kept for some reason?

Reply:
OH!  I forgot!  😄   Thanks for catching that.  I'll delete that commented out line.

Reply:
No worries. Thanks Kevin! 😄